source/_integrations/airos.markdown

```diff
@@ -0,0 +1,112 @@
+---
+title: Ubiquiti airOS
+description: Ubiquiti airOS integration.
+ha_category:
+  - Sensor
+ha_iot_class: Local Polling
+ha_release: 2025.8
+ha_codeowners:
+  - '@CoMPaTech'
+ha_config_flow: true
+ha_domain: airos
+ha_zeroconf: false
+ha_platforms:
+  - sensor
+ha_integration_type: device
+ha_quality_scale: bronze
+---
+
+Ubiquiti's [UISP](https://techspecs.ui.com/uisp/wireless) (Ubiquity Internet Service Provider) product line includes various devices designed for interconnecting locations. This integration provides monitoring capabilities for devices running the airOS opearting system.
+
+There is currently support for the following device types within Home Assistant:
+
+- [Sensor](#sensor)
+
+{% note %}
+Ubiquiti UISP products cannot be managed from their popular [UniFi](/integrations/unifi/) software. They are typically configured using a web browser, the UISP Mobile App, or the UISP Cloud/Self-Hosted platform.
+{% endnote %}
+
+## Prerequisites
+
+This integration only supports devices running airOS 8 and already configured using your browser or the UISP app.
+
+{% include integrations/config_flow.md %}
+
+## Supported devices
+
+### airOS 8
+
+While there is no known limitation to which devices running airOS 8 are supported, success has been reported on:
+
+- PowerBeam 5AC gen2
+- Nanostation 5AC (LOCO5AC) 
+
+## Sensor
+
+This integration exposes the following sensor entities for your airOS devices:
+
+### Network Role
```

Comment:
the titles in this sections: are these the names of the sensors as shown in the UI?

Reply:
Coming from https://www.home-assistant.io/integrations/plugwise/ I preferred the list style, but thought to keep it somewhat in line with the rest of the Ubiquiti family :)

Reply:
Should I rewrite them or are you already committing things? (Let's not clash and waste time)

Reply:
makes sense to keep it consistent. Can't argue with that :-D